tasks/serialize-workers.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+const babel = require('rollup-plugin-babel');
+const resolve = require('rollup-plugin-node-resolve');
+const common = require('rollup-plugin-commonjs');
+const rollup = require('rollup');
+const terser = require('rollup-plugin-terser').terser;
+const fse = require('fs-extra');
+
+async function build(input, {minify = true} = {}) {
+  const plugins = [
+    {
+      name: 'remove export let create',
+      transform(code, id) {
+        if (id !== input) {
+          return null;
+        }
+        return code.replace('export let create;\n', '');
+      }
+    },
+    common(),
+    resolve(),
+    babel({
+      'presets': [
+        [
+          '@babel/preset-env',
+          {
+            'modules': false,
+            'targets': 'last 2 version, not dead'
+          }
+        ]
+      ]
+    }),
+    {
+      name: 'serialize worker and export create function',
+      renderChunk(code) {
+        return `
+          const source = ${JSON.stringify(code)};
+          const blob = new Blob([source], {type: 'application/javascript'});
+          const url = URL.createObjectURL(blob);
+          export function create() {
+            return new Worker(url);
+          }
```

Comment:
We may end up wanting to move the blob and url creation into the `create` function.  Profiling with more representative worker sources should clarify which way to go.